notebooks/benchmark/Default Mask Benchmark.ipynb

```diff
@@ -0,0 +1,283 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "trend_plot(benchmarks)"
+   ]
```

Comment:
what are the x and y axes on the plots? hard to tell.
also is there any way to see the execution time, since I would imagine they would differ most on that?

Reply:
I will add the labels for x and y axes. 
We've left out the runtime metric for the time being because the focus is mostly on the algorithmic metrics. We are considering adding additional performance metrics such as 'runtime' and 'memory' later into an evaluation class, which will also include the sequential perturbation object.